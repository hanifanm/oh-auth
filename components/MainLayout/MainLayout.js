import styles from './MainLayout.scss'
import React from 'react'
import cx from 'classnames'

const MainLayout = ({ className, children }) => {
  return (
    <div className={cx(className, styles.root)}>
      <div className={styles.container}>
        {children}
      </div>
    </div>
  );
}

export default MainLayout;