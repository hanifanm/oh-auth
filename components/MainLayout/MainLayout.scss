.root {
  min-height: 100vh;
  background-color: #f1f1f1;
}

.screen {
  max-width: 400px;
  width: 100%;
  margin: 0 auto;
  background-color: #fff;
  min-height: 100vh;
  padding: 50px 0 100px 0;
}

.container {
  padding: 0 20px;
}
