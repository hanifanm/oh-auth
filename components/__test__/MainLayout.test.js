import { configure, shallow } from 'enzyme'
import React from 'react'
import sinon from 'sinon'
import Adapter from 'enzyme-adapter-react-16'
import MainLayout from '../MainLayout/MainLayout'

configure({ adapter: new Adapter() })

describe('<MainLayout />', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<MainLayout />)
    expect(wrapper).toMatchSnapshot()
  })
})