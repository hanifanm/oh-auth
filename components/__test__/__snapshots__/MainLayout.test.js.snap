// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthLayout /> matches snapshot 1`] = `
<div
  className=""
>
  <div>
    <div>
      <img
        src={
          Object {
            "canInstrument": true,
            "createTransformer": [Function],
            "getCacheKey": [Function],
            "process": [Function],
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`<MainLayout /> matches snapshot 1`] = `
<div
  className=""
>
  <div>
    <div />
  </div>
</div>
`;
